README.md

```diff
@@ -75,6 +75,23 @@ $ gatsby develop
 ```
 6. Open webpage in http://localhost:8000.
 
+## How to Launch the Scraper
+
+```
+% cd data_pipeline/scraper 
+% virtualenv env
+% source env/bin/activate
+
+(env) % python3 -m pip install chromedriver_binary webdriver-manager selenium
```

Comment:
Should by python, not python3 (at least thats the only way that I was able to get it to work).

Also, due to Darren's latest changes, you should include xlrd in the pip install list.

Reply:
Have you tried using python3 in the way I am using it here? I feel it is better to use the later python version and I like that the use of "python3 -m pip" makes it clear what is being used without mucking with shell variables and such.

Reply:
Looks fine for now, I will mess around with this a bit.  I was not able to get it working using the "python3" word in windows, but I will try experiment a bit, and post any changes when I post the windows instructions.